tests/contrib/sqlalchemy/mixins.py

```diff
@@ -24,27 +24,7 @@ class Player(Base):
     name = Column(String(20))
 
 
-class SQLAlchemyTestMixin(object):
```

Comment:
this isn't removed, its just split into a "Base" (setup/teardown/etc without tests) and a Test Mixin (which has tests) in order to support subclassing of the fomer without the latter